### PR TITLE
Fix syntax issues

### PR DESCRIPTION
**Pull request by nicolasgrieco**
- remove the .to_s
 - modify the only_if
 - using variable for php config file path

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@
 librenms_version = node['librenms']['install']['version']
 librenms_file = File.join(librenms_version, '.zip')
```
Comment (jmauro) [inline recipes/default.rb]: You should use ::File.join

File: recipes/default.rb [hunk 2]
```diff
@@ -16,6 +16,7 @@
...
 librenms_archive = File.join(tmpdir, librenms_version)
```
Comment (jmauro) [inline recipes/default.rb]: Same as above

File: recipes/default.rb [hunk 3]
```diff
@@ -16,6 +16,7 @@
...
+librenms_phpconfigfile = File.join(librenms_homedir, 'config.php')
```
Comment (jmauro) [inline recipes/default.rb]: Same as above

File: recipes/default.rb [hunk 4]
```diff
@@ -85,19 +86,19 @@
...
 
   yum_repository 'webtatic' do
-    description (node['librenms']['repo_webtatic']['desc']).to_s
-    baseurl (node['librenms']['repo_webtatic']['url']).to_s
+    description node['librenms']['repo_webtatic']['desc']
+    baseurl node['librenms']['repo_webtatic']['url']
     gpgcheck false
     enabled true
-    only_if { node.normal['librenms']['repo_webtatic']['enabled'] = 'true' }
+    only_if { node['librenms']['repo_webtatic']['enabled'] }
```
Comment (jmauro) [inline recipes/default.rb]: You should only use this attribute for the enabled property:
```
enabled node['librenms']['repo_webtatic']['enabled']
```

File: recipes/default.rb [hunk 5]
```diff
@@ -85,19 +86,19 @@
...
 
   yum_repository 'epel' do
-    description (node['librenms']['repo_epel']['desc']).to_s
-    baseurl (node['librenms']['repo_epel']['url']).to_s
+    description node['librenms']['repo_epel']['desc']
+    baseurl node['librenms']['repo_epel']['url']
     gpgcheck false
     enabled true
-    only_if { node.normal['librenms']['repo_epel']['enabled'] = 'true' }
+    only_if { node['librenms']['repo_epel']['enabled'] }
```
Comment (jmauro) [inline recipes/default.rb]: Same as above

**Review by jmauro — CHANGES_REQUESTED** [review]

File: recipes/default.rb (hunk 1 above)
Comment (nicolasgrieco) [reply recipes/default.rb]: done

File: recipes/default.rb (hunk 4 above)
Comment (nicolasgrieco) [reply recipes/default.rb]: done.

File: recipes/default.rb (hunk 5 above)
Comment (nicolasgrieco) [reply recipes/default.rb]: done

Comment (jmauro) [reply recipes/default.rb]: You should use `yum-epel`

**Review by jmauro — CHANGES_REQUESTED** [review]

File: recipes/default.rb (hunk 2 above)
Comment (nicolasgrieco) [reply recipes/default.rb]: done

File: recipes/default.rb (hunk 3 above)
Comment (nicolasgrieco) [reply recipes/default.rb]: done

File: recipes/default.rb [hunk 6]
```diff
@@ -85,19 +86,17 @@
...
 
   yum_repository 'epel' do
-    description (node['librenms']['repo_epel']['desc']).to_s
-    baseurl (node['librenms']['repo_epel']['url']).to_s
+    description node['librenms']['repo_epel']['desc']
+    baseurl node['librenms']['repo_epel']['url']
     gpgcheck false
-    enabled true
-    only_if { node.normal['librenms']['repo_epel']['enabled'] = 'true' }
+    enabled node['librenms']['repo_epel']['enabled']
```
Comment (jmauro) [inline recipes/default.rb]: I still don't understand why you are not using `yum-epel`

Comment (nicolasgrieco) [reply recipes/default.rb]: Fixed.

File: recipes/default.rb [hunk 7]
```diff
@@ -64,6 +65,9 @@
   librenms_phpconf = '/etc/php.d/librenms.ini'
   rrdcached_config = '/etc/sysconfig/rrdcached'
 
+  include_recipe 'yum-epel'
+  node.default['yum']['epel']['enabled'] = true
```
Comment (jmauro) [inline recipes/default.rb]: @nicolasglab you should add this before before including the the recipe (Better in the cookbook attribute)

**Review by jmauro — CHANGES_REQUESTED** [review]

File: recipes/default.rb (hunk 7 above)
Comment (nicolasgrieco) [reply recipes/default.rb]: done.

**Review by jmauro — APPROVED** [review]

File: recipes/default.rb [hunk 8]
```diff
@@ -85,19 +88,10 @@
...
-    enabled true
-    only_if { node.normal['librenms']['repo_webtatic']['enabled'] = 'true' }
-  end
-
-  yum_repository 'epel' do
-    description (node['librenms']['repo_epel']['desc']).to_s
-    baseurl (node['librenms']['repo_epel']['url']).to_s
+    description node['librenms']['repo_webtatic']['desc']
+    baseurl node['librenms']['repo_webtatic']['url']
     gpgcheck false
```
Comment (jmauro) [inline recipes/default.rb]: Why don't enforce the gpgp keys? They are present on teh repository: https://repo.webtatic.com/yum/

**Review by jmauro — CHANGES_REQUESTED** [review]

File: recipes/default.rb (hunk 8 above)
Comment (nicolasgrieco) [reply recipes/default.rb]: Done!

File: recipes/default.rb [hunk 9]
```diff
@@ -85,19 +88,11 @@
...
-
-  yum_repository 'epel' do
-    description (node['librenms']['repo_epel']['desc']).to_s
-    baseurl (node['librenms']['repo_epel']['url']).to_s
-    gpgcheck false
-    enabled true
-    only_if { node.normal['librenms']['repo_epel']['enabled'] = 'true' }
+    description node['librenms']['repo_webtatic']['desc']
+    baseurl node['librenms']['repo_webtatic']['url']
+    gpgcheck true
```
Comment (jmauro) [inline recipes/default.rb:93]: FAR: Maybe put this is as an attribute

**Review by jmauro — APPROVED** [review]

**Pull request merged**